.github/workflows/deployment.yml

```diff
@@ -19,100 +19,118 @@ jobs:
-      - name: Environment Canada HRDPS cronjob (Donald)
-        uses: bcgov/openshift-launchpad-deployment@master
-        with:
-          AUTH_TOKEN: ${{ secrets.OC_DEV_TOKEN_EDIT }}
-          SCRIPT: openshift/scripts/oc_provision_ec_hrdps_cronjob.sh ${SUFFIX} apply
-
-      - name: Environment Canada RDPS cronjob (Donald)
-        uses: bcgov/openshift-launchpad-deployment@master
-        with:
-          AUTH_TOKEN: ${{ secrets.OC_DEV_TOKEN_EDIT }}
-          SCRIPT: openshift/scripts/oc_provision_ec_rdps_cronjob.sh ${SUFFIX} apply
-
-      # f.y.i.: ZAP Scan must be able to log an issue or it will fail.
-      - name: ZAP Scan
-        uses: zaproxy/action-baseline@v0.4.0
-        with:
-          target: "https://wps-pr-${{ github.event.number }}.pathfinder.gov.bc.ca"
-          rules_file_name: ".zap/rules.tsv"
-          # Do not return failure on warnings - TODO: this has to be resolved!
-          cmd_options: "-I"
+          AUTH_TOKEN: ${{ secrets.OC4_DEV_TOKEN }}
+          SCRIPT: PVC_SIZE=2Gi PROJ_DEV=e1e498-dev IMAGE_NAMESPACE=e1e498-tools IMAGE_SERVER=image-registry.openshift-image-registry.svc:5000 IMAGE_VERSION=v11 EPHEMERAL_STORAGE=True openshift/scripts/oc_provision_db.sh ${SUFFIX} apply
+
+  # prepare-database:
+  #   name: Prepare Dev Database
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - name: Set Variables
+  #       shell: bash
+  #       run: |
+  #         echo "SUFFIX=pr-${{ github.event.number }}" >> $GITHUB_ENV
+
+  #     - name: Checkout
+  #       uses: actions/checkout@v2
+
+  #     - name: Deploy PostGIS instance
+  #       uses: bcgov/openshift-launchpad-deployment@v1.1
+  #       with:
+  #         AUTH_TOKEN: ${{ secrets.OC_DEV_TOKEN_EDIT }}
+  #         SCRIPT: EPHEMERAL_STORAGE=True openshift/scripts/oc_provision_db.sh ${SUFFIX} apply
+
+  # prepare-database-backups:
+  #   name: Prepare Dev Database Backups
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - name: Set Variables
+  #       shell: bash
+  #       run: |
+  #         echo "SUFFIX=pr-${{ github.event.number }}" >> $GITHUB_ENV
+
+  #     - name: Checkout
+  #       uses: actions/checkout@v2
+
+  #     - name: Prepare Matomo Backup Volume PVC
+  #       uses: bcgov/openshift-launchpad-deployment@v1.1
+  #       with:
+  #         AUTH_TOKEN: ${{ secrets.OC_DEV_TOKEN_EDIT }}
+  #         SCRIPT: openshift/scripts/oc_provision_matomo_backup_pvc.sh ${SUFFIX} apply
+
+  #     - name: Prepare Matomo Backup Deployment Config
+  #       uses: bcgov/openshift-launchpad-deployment@v1.1
+  #       with:
+  #         AUTH_TOKEN: ${{ secrets.OC_DEV_TOKEN_EDIT }}
+  #         SCRIPT: openshift/scripts/oc_provision_matomo_backup.sh ${SUFFIX} apply
+
+  #     - name: Prepare Matomo Backup Cronjob
+  #       uses: bcgov/openshift-launchpad-deployment@v1.1
+  #       with:
+  #         AUTH_TOKEN: ${{ secrets.OC_DEV_TOKEN_EDIT }}
+  #         SCRIPT: openshift/scripts/oc_provision_matomo_backup_cronjob.sh ${SUFFIX} apply
+
+  # build-and-deploy:
```

Comment:
You don't want to deploy to ocp3 anymore?

Reply:
@Kyubinhan this PR should be in draft. That's just me debugging stuff.